packages/core/src/actors.ts

```diff
@@ -504,25 +391,98 @@ export function createBehaviorFrom<TEvent extends EventObject>(
 ): Behavior<TEvent, undefined>;
 export function createBehaviorFrom(entity: Spawnable): Behavior<any, any> {
```

Comment:
as mentioned somewhere - this should be removed for now because we should only accept lazy entities and with promises we can't detect what's enclosed in the function, an "aggregate" factory-like this would be nice but it would require some implementation adjustments to be made - in the current form this is just not meeting the requirements

Reply:
Removed